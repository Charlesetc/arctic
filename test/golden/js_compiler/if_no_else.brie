define main [
  if true [
    3
  ]
]
