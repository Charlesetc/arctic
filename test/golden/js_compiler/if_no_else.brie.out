const global_functions = {1: {}};
function main() {
	if (true) {3;} else {} ; return __unit;
}
console.log(main());
var test/golden/js_compiler/if_no_else.brie_main = new_closure(1)